e2e-tests/tests/book.spec.ts

```diff
@@ -19,4 +19,13 @@ test.describe("tests:", () => {
       page.locator('[id="BookMeetingSection"]').first()
     ).toBeVisible();
   });
+
+  test("navigating to visit using waffle menu", async ({ page }) => {
+    const waffle = page.locator('[data-icon-name="Waffle"]').first();
+    await waffle.click();
+
+    // Click text=Visit
+    await page.locator("text=Visit").click();
+    await expect(page).toHaveURL("http://localhost:8080/visit");
```

Comment:
lets use the constant SERVER_URL here

Reply:
Will make these changes ina subsequent PR to avoid resetting current votes (again)